release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+                kibana
+                logging
+                logging_index_time
+                memory_availability
+                ovs_version
+                package_availability
+                package_update
+                package_version
+
+$ ansible-playbook playbooks/byo/openshift-checks/adhoc.yml -e
+openshift_checks=fluentd_config,logging_index_time,docker_storage
+----
+
+Alternatively, they are included in the health playbook:
+
+----
+$ ansible-playbook playbooks/byo/openshift-checks/health.yml
+----
+
+To capture the output:
+
+----
+$ ansible-playbook playbooks/byo/openshift-checks/health.yml -e
+openshift_checks_output_dir=/tmp/checks
+----
+
+[[ocp-37-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-37-journald-system-logs]]
+==== Jouranld for System Logs and JSON File for Container Logs
+
+Docker log driver is set to `json-fiile` as the default for all nodes. Docker
+`log-driver` can be set to `journal`, but there is no log rate throttling with
+journal driver. So, there is always a risk for denial-of-service attacks from
+rogue containers.
+
+Fluentd will automatically determine which log driver (`journald` or
+`json-file`) the container runtime is using. Fluentd will now always read logs
+from journald and also *_/var/log/containers_* (if `log-driver` is set to
+`json-file`). Fluentd will no longer read from *_/var/log/messages_*.
+
+See
+xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating
+Container Logs] for more information.
+
+[[ocp-37-docker-events-and-api-calls-aggregated-to-efk-as-logs]]
+==== Docker Events and API Calls Aggregated to EFK as Logs
+
+Fluentd captures standard error and standard out from the running containers on
+the node. With this change, fluentd collects all the errors and events coming
+from the docker daemon running on the node and sends it to Elasticsearch (ES).
+
+Enable this via the {product-title} installer:
+
+----
+openshift_logging_fluentd_audit_container_engine=true
+----
+
+The collected information is in operation indices of ES and only cluster
+administrators have visual access. The event message includes action, pod name,
+image name, and user time-stamp.
+
+[[ocp-37-master-events-aggregated-to-efk-as-logs]]
+==== Master Events are Aggregated to EFK as Logs
+
+The *eventrouter* pod scrapes the events from kubernetes API and and outputs to
+*STDOUT*. The *fluentd* plug-in transforms the log message and sends it to
+Elasticsearch (ES).
+
+Enable `openshift_logging_install_eventrouter` by setting it to `true`. It is
+off by default. *Eventrouter* is deployed to the default namespace. Collected
+information is in operation indices of ES and only cluster administrators have
+visual access.
+
+See the
+link:https://github.com/openshift/origin-aggregated-logging/blob/master/docs/proposals/kube_events_design_doc.md[design
+documentation] for more information.
+
+==== Kibana Dashboards for Operations Are Now Shareable
```

Comment:
I don't feel the linked section coves what this note try's to cover, can we get a more detailed example?